packages/flutter/test/material/theme_defaults_test.dart

```diff
@@ -183,4 +183,59 @@ void main() {
     });
   });
 
+  group('FloatingActionButton', () {
+    testWidgets('theme: ThemeData.light(), enabled: true', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData.light(),
+          home: Center(
+              child: FloatingActionButton(
+                onPressed: () { }, // button.enabled == true
+                child: const Icon(Icons.add),
+              )
+          ),
+        ),
+      );
+
+      final FloatingActionButton raw = tester.widget<FloatingActionButton>(find.byType(FloatingActionButton));
```

Comment:
Test the RawMaterialButton as in the next test.

Reply:
Done. I don't know what nonsense I uploaded before.